packages/react-examples/package.json

```diff
@@ -52,6 +52,7 @@
     "@fluentui/react-make-styles": "^9.0.0-alpha.56",
     "@fluentui/react-menu": "^9.0.0-alpha.63",
     "@fluentui/react-shared-contexts": "^9.0.0-alpha.21",
+    "@fluentui/react-switch": "9.0.0-alpha.0",
```

Comment:
I don't think we need to add this over here since we are collocating our examples in their respective packages.